### ISSUE TITLE
Duplicate valid detections

### ISSUE THREAD

**Issue by jkburges** [aodn/aatams#115]
I've discovered this whilst trying to create a unique index on `timestamp, receiver_name, transmitter_id` (which should indeed be unique) - there are some duplicates though.

e.g.

```sql
select count(*) from valid_detection
where timestamp = '2013-06-13 21:18:50+00'
and receiver_name = 'VR2W-113916'
and transmitter_id = 'A69-1601-28325'
```

gives `2` instead of the expected `1` (or `0` if there's no match).

**Comment by ktattersall** [comment]
Do you know if this issue been resolved by #134 and/or  #136, @jkburges ?

**Comment by jkburges** [comment]
> Do you know if this issue been resolved by #134 and/or #136, @jkburges ?

Sorry, only just saw this now. This issue has been made redundant/invalid by https://github.com/aodn/aatams/pull/216 and related changes.
